static/script-tests/tests/devices/mediaplayer/cehtmlcommontests.js

```diff
@@ -346,6 +346,20 @@ window.commonTests.mediaPlayer.cehtml.mixinTests = function (testCase, mediaPlay
         });
     };
 
+    mixins.testBeginPlaybackFromSetsCurrentTimeAndCallsPlayOnMediaElementWhenInStoppedState = function(queue) {
```

Comment:
@sdclibbery rename to playable is on a branch I've been working on - there may be some tidy up needed to merge both that one and this.